app/mailers/course_registration_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CourseRegistrationMailer < ApplicationMailer
+  def new_registration(registrations)
+    @registrations = registrations
+    mail(
+      to: registrations.first.student.email,
+      subject: "New School Swing registration for #{registrations.first.course.meeting_dates.first.third.strftime("%B")} classes!"
```

Comment:
Line is too long. [130/80]<br>Prefer single-quoted strings inside interpolations.